src/InfoStructHelper.cc

```diff
@@ -354,12 +357,36 @@ namespace mu2e {
       tchinfo.csize = cc->size();
       tchinfo.edep = cc->energyDep();
       tchinfo.edeperr = cc->energyDepErr();
+      tchinfo.cposX = cc->cog3Vector().getX();
```

Comment:
What's the difference between this and ```tch._cpos``` which is used for ```tchinfo.poca```? Are they representing different things?

Reply:
cpos is the cluster centroid position the poca vector is the point of closest approach of the track to the cluster centroid, their difference is the doca.